src/toil/test/utils/utilsTest.py

```diff
@@ -79,14 +80,27 @@ def testAWSProvisionerUtils(self):
         clusterName = 'cluster-utils-test' + str(uuid.uuid4())
         try:
             system([self.toilMain, 'launch-cluster', '--nodeType=t2.micro', '--keyPairName=jenkins@jenkins-master',
-                 clusterName, '--provisioner=aws'])
+                    clusterName, '--provisioner=aws'])
         finally:
             system([self.toilMain, 'destroy-cluster', '--provisioner=aws', clusterName])
         try:
+            from toil.provisioners.aws.awsProvisioner import AWSProvisioner
             # launch preemptable master with same name
             system([self.toilMain, 'launch-cluster', '--nodeType=m3.medium:0.2', '--keyPairName=jenkins@jenkins-master',
                     clusterName, '--provisioner=aws', '--logLevel=DEBUG'])
             system([self.toilMain, 'ssh-cluster', '--provisioner=aws', clusterName])
+            compareTo = "import sys; assert sys.argv[1]=='testString'"
+            AWSProvisioner.sshLeader(clusterName=clusterName,
+                                     args=['python', '-', 'testString'],
+                                     input=compareTo)
+            compareTo = "import sys; assert sys.argv[1]=='  testString'"
+            AWSProvisioner.sshLeader(clusterName=clusterName,
+                                     args=['python', '-', '  testString'],
+                                     input=compareTo)
+            compareTo = "import sys; assert sys.argv[1]=='$PATH'"
+            AWSProvisioner.sshLeader(clusterName=clusterName,
+                                     args=['python', '-', '$PATH'],
+                                     input=compareTo)
```

Comment:
This could be a loop that computes `compareTo` dynamically using `%r`.

I would also test

```
args=['python', '-', '"testString"']
args=['python', '-', '"'] # one double quote character
args=['python', '-', "`"] # one single quote character
args=['python', '-', '\\'] # one single single backslash character
```